data/changelog.txt

```diff
@@ -380,6 +381,19 @@ Fix Brackenwight special attack message
 [Magiclysm] Last Mage on Earth start
 Tamable Wildlife: add Cougars to list of tameables
 [Magiclysm] Add spellcasting proficiencies.
+[MoM] High levels of weariness turn off your powers
+More specific messages for being unable to use supernatural powers
+[Xedra Evolved] New Arvore spell--Verdant Imprisonment
+Tune channeling spells with scale modifier.
+[MoM] Edit portal storm conditions to support moddability, use that for Telepathic Shield in MoM
+allow incandescent lance to repair metal items
+XE: Trait to avoid learning dream magick
+Too much pain shuts down your powers
+Tune evocation spells with scale modifier.
+[MoM] Add luminance to Eaters
+[MoM] Add Noisemaker telekinetic power
+[MoM] Remove extended channeling recipe after learning, add Thought-Form Construction practice recipe
+[MoM] Feral telekinetics can't throw you with telekinesis if you have Inertial Barrier up and vice versa
```

Comment:
```suggestion
Tune channeling spells with scale modifier.
[MoM] Edit portal storm conditions to support moddability, use that for Telepathic Shield in MoM
XE: Trait to avoid learning dream magick
```